gbpservice/neutron/tests/unit/nfp/configurator/agents/test_lb_agent.py

```diff
@@ -376,43 +413,36 @@ def _test_handle_event(self, rule_list_info=True):
 
         """
 
-        agent = self._get_lb_handler_objects()
+        agent = self._get_lb_handler_objects(sc, self.drivers, rpcmgr)
         driver = lb_driver.HaproxyOnVmDriver()
 
-        with mock.patch.object(
-                agent.plugin_rpc, 'get_logical_device') as (
-                mock_get_logical_device), \
-                mock.patch.object(
-                    agent.plugin_rpc, 'update_status') as (mock_update_status), \
-                mock.patch.object(
-                    agent.plugin_rpc, 'update_pool_stats') as (mock_update_pool_stats),\
-                mock.patch.object(agent, '_get_driver', return_value=driver), \
-                mock.patch.object(
+        with mock.patch.object(agent, '_get_driver', return_value=driver), \
```

Comment:
Still there are lot of "\" code separators. Why are these not removed after mentioning so many times in review comments? Code will not be readable with this. Need to correct all of the code immediately.